src/sciwrappers.c

```diff
@@ -964,6 +968,12 @@ void sci_set_font(ScintillaObject *sci, gint style, const gchar *font, gint size
 }
 
 
+gint sci_get_fontsize(ScintillaObject *sci, gint style)
+{
+	return (gint) SSM(sci, SCI_STYLEGETSIZE, (uptr_t) style, 0);
+}
```

Comment:
I would still like to see the function renamed with an underscore between `font` and `size`.

Also, we now are using [SCI_STYLESETSIZEFRACTIONAL](https://scintilla.org/ScintillaDoc.html#SCI_STYLESETSIZEFRACTIONAL), so this should probably be using fractional sizes as well and return a `gdouble`.

Something like that (untested)
```c
gdouble sci_get_font_size(ScintillaObject *sci, gint style)
{
	return SSM(sci, SCI_STYLEGETSIZEFRACTIONAL, (uptr_t) style, 0) / (gdouble) SC_FONT_SIZE_MULTIPLIER;
}
```

Reply:
Changed and tested. Now looks like your code snippet below.